### PR TITLE
Fixes SMES hacking

### PR DESCRIPTION
**Pull request by FlamingLily**
~~In preparation for my grand political strategy to get SMES hacking wires added to the aurora wiki, I made sure they were working.~~

It's on the wiki now so you have to merge this it's canon 🫡 

Changes:
* The RCON wire in a SMES now actually disconnects a SMES from RCON (before it only prevented AI control).
* When the input wire is cut in a SMES, it will now correctly report that the SMES is not charging on the UI.
* The Failsafe and Grounding wires of a SMES now correctly trigger the red light when only one is cut, instead of both needing to be triggered.

### CODE REVIEW

**Comment by FlamingLily** [comment]
!bugfix

**Comment by FlamingLily** [comment]
It might be an idea to replace the "green red blue" lights with more descriptive names:

The connection light
The safety warning light
The remote control light

If this is desired I can do that.

File: code/modules/power/smes.dm [hunk 1]
```diff
@@ -224,8 +224,10 @@
 
 	var/goal = (delta_power < 0) ? (charge) : (capacity - charge)
 	time = world.time + (delta_power ? ((goal / abs(delta_power)) * (world.time - last_time)) : 0)
-	// If it is negative - we are discharging
-	if(delta_power < 0)
+
+	if(input_cut == TRUE) // Cannot charge if input wire cut
```
Comment (NonQueueingMatt) [inline code/modules/power/smes.dm:228]: if this is a boolean then check it this way:
`if(input_cut)`

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
!review

File: code/modules/power/smes.dm (hunk 1 above)
Comment (FlamingLily) [reply code/modules/power/smes.dm:228]: oh duh right
```suggestion
	if(input_cut) // Cannot charge if input wire cut
```

File: code/modules/modular_computers/file_system/programs/engineering/rcon.dm [hunk 2]
```diff
@@ -21,17 +21,18 @@
...
-		"input_set" = SMES.input_attempt,
-		"input_val" = round(SMES.input_level),
-		"output_set" = SMES.output_attempt,
-		"output_val" = round(SMES.output_level),
-		"input_level_max" = SMES.input_level_max,
-		"output_level_max" = SMES.output_level_max,
-		"output_load" = round(SMES.output_used),
-		"RCON_tag" = SMES.RCon_tag
-		)))
+		if(SMES.RCon == TRUE)
```
Comment (FlamingLily) [inline code/modules/modular_computers/file_system/programs/engineering/rcon.dm:24]:
```suggestion
		if(SMES.RCon)
```

**Review by Arrow768 — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

**Pull request merged**